src/components/common/AddEditOrgPerson.vue

```diff
@@ -49,10 +49,9 @@
           cols="12"
           sm="9"
         >
-          <v-card
+          <MessageBox
             v-if="isCompletingParty && !isRoleStaff && isTypeCoop"
-            outlined
-            class="message-box rounded-0"
+            color="gold"
```

Comment:
Example:
![image](https://github.com/bcgov/business-create-ui/assets/10002889/124e9a80-fed0-43a5-83df-607be532abd3)

Reply:
I was thinking to maybe have the text as a prop for it. But looking now, this actually makes it so much more flexible since we can have whatever we want here in the parent and inside the component. Sweet!